minimize.sh

```diff
@@ -0,0 +1,9 @@
+uglifyjs ../../../../javascript/drawing.js -o ../../../../javascript/drawing_min.js -m -c hoist_vars=true & uglifyjs ../../../../javascript/AMhelpers2.js -o ../../../../javascript/AMhelpers2_min.js -m -c hoist_vars=true &
```

Comment:
There already was a buildmin.sh script for this purpose.  In particular, this script references your specific directories, so isn't generally useful.